packages/cli/src/cli.ts

```diff
@@ -13,14 +13,15 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
-import { EOL } from 'os'
+import * as os from 'os'
```

Comment:
nit: seems like the more common way to import this in our code is
```suggestion
import os from 'os'
```